src/condor_utils/docker-api.cpp

```diff
@@ -1141,18 +1133,17 @@ int DockerAPI::inspect( const std::string & containerID, ClassAd * dockerAd, Con
 		return -1;
 	inspectArgs.AppendArg( "inspect" );
 	inspectArgs.AppendArg( "--format" );
-	StringList formatElements(	"ContainerId=\"{{.Id}}\" "
-								"Pid={{.State.Pid}} "
-								"Name=\"{{.Name}}\" "
-								"Running={{.State.Running}} "
-								"ExitCode={{.State.ExitCode}} "
-								"StartedAt=\"{{.State.StartedAt}}\" "
-								"FinishedAt=\"{{.State.FinishedAt}}\" "
-								"DockerError=\"{{.State.Error}}\" "
-								"OOMKilled=\"{{.State.OOMKilled}}\" " );
-	char * formatArg = formatElements.print_to_delimed_string( "\n" );
+	const std::string formatArg("ContainerId=\"{{.Id}}\"\n"
```

Comment:
As long as we are making changes here, how about using a c++ 11 raw string literal ?